ckanext/csrf_filter/test_anti_csrf.py

```diff
@@ -248,6 +249,21 @@ def _check_config(self, secret_key, secure_cookies=False,
         self.assertEqual(anti_csrf.token_expiry_age, token_expiry_age)
         self.assertEqual(anti_csrf.token_renewal_age, token_renewal_age)
 
+    def test_exempt_rules(self):
+        """ Tests that the custom exempt rules pass their conditions
```

Comment:
This is not very descriptive. Perhaps "Test that requests matching the exemption rules are not checked for tokens" or something similar.